kiwiapp-public/tailwind.config.ts

```diff
@@ -83,13 +84,55 @@ const config: Config = {
         plum: "#872B58",
         beige: "#F0EEEA",
         sunset: "#EA8F58",
+
+        // shadcn-ui Tailwind Configuration
+        border: "hsl(var(--border))",
+        input: "hsl(var(--input))",
+        ring: "hsl(var(--ring))",
+        background: "hsl(var(--background))",
+        foreground: "hsl(var(--foreground))",
```

Comment:
Let's not include these. For Shadcn components we should only take the component logic and not take any of its styles.